### DISCUSSION TITLE
A big thanks

### DISCUSSION THREAD

**Discussion by kandreyc** [DaZombieKiller/UnityRoslynUpdater#7] [General]
Hi, wanted to say a big thank you for your package. I've searched for something like this for a long time!

Hope it's fine that I [borrowed](https://github.com/kandreyc/unity-csharp-patch) your solution and improved it a bit to fit my needs. The main goal was to allow using it per ``.asmdef`` and be available on any platform.

Thank you for your work!

**Comment by DaZombieKiller** [comment]
> Hope it's fine that I [borrowed](https://github.com/kandreyc/unity-csharp-patch) your solution and improved it a bit to fit my needs.

I am more than okay with this, that's why it's open source after all! :)

Your project looks very cool, it takes the idea a step further which I like quite a lot.
